packages/sourcecred/src/cli/grain.js

```diff
@@ -45,6 +45,16 @@ const grainCommand: Command = async (args, std) => {
 
   const {distributions, ledger} = await grain(grainInput);
 
+  const grainIntegrationResults = executeGrainIntegrationsFromGrainInput(
+    grainInput,
+    ledger,
+    distributions
+  );
```

Comment:
This is related to my other comment.
```suggestion
  const {distributions, ledger: ledgerBeforeIntegrations} = await grain(grainInput);

  const {ledger, results: grainIntegrationResults} = executeGrainIntegrationsFromGrainInput(
    grainInput,
    ledgerBeforeIntegrations,
    distributions
  );
```